### ISSUE TITLE
A11y_Azure Data Studio Nov2023_Files_Add Database File_ScreenReader: Name is not defined descriptively for Path section "More Option" button.

### ISSUE THREAD

**Issue by Swati1700** [microsoft/azuredatastudio#25212]
"Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility or the Divisional Driver."
[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier.

GitHub Tags:
#A11yTCS; #A11yMAS; #A11ySev2; #DesktopApp; #WCAG4.1.2; #NameRoleValue; #FTP; #SH_Azure Data Studio_Nov2023_Linux_Dec2023; #SQL Azure Data Studio; #Linux; #ORCA;

Environment Details:
Application Name: Azure Data Studio
OS Name: Ubuntu 22.04.2 LTS
Azure data studio 1.47.0
Screen Reader: ORCA

### Repro Steps:

1. Install Azure data studio application and open it
2. Make sure to enable "Preview" feature in settings.
3. Turn on ORCA.
4. Tab till "Database" and press ENTER to expand the tree.
5. Tab till any database and pressF10 to open context menu.
6. Tab till "Properties Preview" and press ENTER.
7. Tab till "Files" tab and press ENTER.
8. Tab till "Database files" table and press ENTER to select the row.
9. Tab till "Add" button and press ENTER.
10. "Add Database File" dialog will appear.
11. Tab till " more option" button in Path Section.
12. You will observe that voice over is announcing as "..., button" for more option button.

### Actual Result:
ORCA is announcing as "..., button" for more option button present in "Path" section of "Add Database File" dialog.

**Note: Similar issue is observed for Edit Database File ->Path section "More Option" button.**

### Expected Result:
ORCA should announce as "Path More option button" for button present in "Path" section of "Add Database File" dialog. The name for "more option" button should be described clearly to understand the purpose of control.

### User Impact:
Screen Reader users will face difficulties to understand the purpose of the button if name is not descriptive enough.

Attachment:
[25212.zip](https://github.com/microsoft/azuredatastudio/files/13948837/25212.zip)

**Comment by barbaravaldez** [comment]
Addressed by https://github.com/microsoft/azuredatastudio/pull/25233

**Comment by corivera** [comment]
@Swati1700 This has been fixed in the latest Ubuntu Insiders build of ADS [here](https://azuredatastudio-update.azurewebsites.net/latest/linux-deb-x64/stable). Can you verify that the issue is fixed?

**Comment by Swati1700** [comment]
Verified the issue in the above build Version: 1.47.1
Issue is still reproducing, please find the repro attachment below.
[Repro.zip](https://github.com/microsoft/azuredatastudio/files/14111110/Repro.zip)

**Comment by kisantia** [comment]
@Swati1700 can you please try testing with the [latest insiders build](https://github.com/microsoft/azuredatastudio?tab=readme-ov-file#try-out-the-latest-insiders-build-from-main-branch)? The build you tried did not yet have the fix.

**Comment by Swati1700** [comment]
Verified the issue in the below environment Version: 1.49.0- insider 
Issue is fixed and does not repro. Please find the closing attachment below.

https://github.com/microsoft/azuredatastudio/assets/86710888/49997300-46a4-41c5-8403-1ef097c281cc
